Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -604,9 +610,10 @@ inline void SeedFinder<external_spacepoint_t, platform_t>::filterCandidates(
         cotThetaB = -zB * std::sqrt(iDeltaRB2);
         cotThetaT = zT * std::sqrt(iDeltaRT2);
 
-        rMxy = std::hypot(rMTransf[0], rMTransf[1]);
-        float Ax = rMTransf[0] / rMxy;
-        float Ay = rMTransf[1] / rMxy;
+        rMxy = std::sqrt(rMTransf[0] * rMTransf[0] + rMTransf[1] * rMTransf[1]);
```

Comment:
```suggestion
        rMxy = std::hypot(rMTransf[0], rMTransf[1]);
```

I guess `Acts::VectorHelpers::perp` might also be an option here if this is an Eigen object